src/renderer/components/AgentView/EventItem/index.tsx

```
@@ -20,6 +35,25 @@ export function EventItem({ event, isLatest }: EventItemProps) {
     case 'step_error':
       return <StepEvent event={event} isLatest={isLatest} />;
 
+    case 'input_required':
+      if (!sessionId || !onInputSubmit || !onInputCancel) {
+        return null;
+      }
+      const requestId = event.data.requestId;
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)